tests/base/test_Objectivefct.py

```diff
@@ -41,6 +41,16 @@ def test_derivs(self):
         objfct = ObjectiveFunction.L2ObjectiveFunction()
         self.assertTrue(objfct.test(eps=1e-9))
 
+    def test_deriv2(self):
```

Comment:
Thanks @lheagy !

Did this actually fail before?

Reply:
The deriv2 was tested in the test above and it passed because it was passing a vector around (internally, we always pass `v`, so the correct deriv2 was being provided. This test just makes sure that the deriv2 when you provide a vec is the same as if you don't (and this failed before)